Ginger/GingerCoreNET/RecordingLib/POMObjectHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Amdocs.Ginger.Repository;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amdocs.Ginger.CoreNET
+{
+    public class POMObjectRecordingHelper
+    {
+        public ApplicationPOMModel ApplicationPOM { get; set; }
```

Comment:
how this POMObjectRecordingHelper is helping? anyway you pull the title/URL directly from POM object so for what thiw wrapper class is needed?

Reply:
this wrapper is just used in "RecordingManager" to hold the list of all the POMs, when page is changed while recording so used this only in recordingmanager class